src/test/java/com/github/bfh/study/slb/imports/parser/PartialXmlParserTest.java

```diff
@@ -56,4 +59,20 @@ public void testParserWithoutElements() throws Exception {
         parser.close();
         assert count == 0 : "the count value should be zero";
     }
+
+    public void testEuListParsing() throws Exception {
```

Comment:
@samsamann, why do you don't use the context? and the EuSanctionsList.java as in the method testParsing()?

Reply:
testParsing() tests only if the JAXB annotations are correct. testEuListParsing is an integration test and uses the original xml file.

Reply:
OK